tests/integration/docker_utils/test_docker.py

```diff
@@ -869,7 +945,7 @@ def test_get_container_ip(self, docker_client: ContainerClient, dummy_container)
         docker_client.start_container(dummy_container.container_id)
         ip = docker_client.get_container_ip(dummy_container.container_id)
         assert re.match(
```

Comment:
Could be useful as a reusable utility function `is_ipv4_address(..)` in either `common.py` or `testutil.py` 👍